src/main/java/edu/unc/lib/boxc/migration/cdm/DestinationsCommand.java

```diff
@@ -48,6 +48,37 @@ public class DestinationsCommand {
     public int generate(@Mixin DestinationMappingOptions options) throws Exception {
         long start = System.nanoTime();
 
+        if (options.isArchivalCollections()) {
```

Comment:
just for clarify, i'd suggest moving everything inside this conditional into a private method which gets called here, so here it'd be
```
if (options.isArchivalCollections()) {
    return generateArchivalCollectionMappings(options);
} else {
    return generateBasicMappings(options);
}
```
other something along those lines